examples/device-dashboard/Makefile

```diff
@@ -1,16 +1,16 @@
-PROG ?= example                   # Program we are building
-PACK ?= ./pack                    # Packing executable
-DELETE = rm -rf                   # Command to remove files
-OUT ?= -o $(PROG)                 # Compiler argument for output file
-SOURCES = main.c mongoose.c net.c packed_fs.c      # Source code files
-CFLAGS = -W -Wall -Wextra -g -I.  # Build options
+PROG ?= ./example       # Program we are building
+PACK ?= ./pack          # Packing executable
+DELETE = rm -rf         # Command to remove files
+OUT ?= -o $(PROG)       # Compiler argument for output file
+SOURCES = main.c mongoose.c net.c       # Source code files
+CFLAGS = -W -Wall -Wextra -g -I.        # Build options
 
 # Mongoose build options. See https://mongoose.ws/documentation/#build-options
-CFLAGS_MONGOOSE += -DMG_ENABLE_PACKED_FS=1
+CFLAGS_MONGOOSE +=
 
 FILES_TO_EMBED ?= $(wildcard web_root/*)
 
```

Comment:
I guess this macro is not used anymore and can be removed

Reply:
Yes! Can be removed